src/glados/utils/automated_install.py

```
@@ -0,0 +1,129 @@
+        print("Ollama is not installed")
+        if sys.platform == "win32":
+            print("Installing Ollama on Windows...")
+            install_ollama_windows()
+        elif sys.platform == "linux":
+            print("Installing Ollama on Linux...")
+            install_ollama_linux()
+        elif sys.platform == "darwin":
+            print("Installing Ollama on macOS...")
+            install_ollama_macos()
+        else:
+            raise RuntimeError("Unsupported platform. This script supports Windows, Linux, and macOS.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)